test/built-ins/Proxy/setPrototypeOf/internals-call-order.js

```diff
@@ -41,7 +41,7 @@ var proxy = new Proxy(target, {
   }
 });
 
-Reflect.setPrototypeOf(proxy, proto);
+assert.sameValue(Reflect.setPrototypeOf(proxy, proto), false);
```

Comment:
shouldn't this return 'true' given that the prototype argument is the original one?